packages/crypto-block/source/serializer.ts

```diff
@@ -137,6 +145,30 @@ export class Serializer implements Contracts.Crypto.IBlockSerializer {
 		});
 	}
 
+	public async serializeLockProof(lockProof: Contracts.Crypto.IBlockLockProof): Promise<Buffer> {
+		return this.serializer.serialize<Contracts.Crypto.IBlockLockProof>(lockProof, {
+			length: this.lockProofSize(),
+			skip: 0,
+			schema: {
+				signature: {
+					type: "consensusSignature",
+				},
+				validators: {
+					type: "validatorSet",
+				},
+			},
+		});
+	}
+
+	public async serializeProposed(proposedBlock: Contracts.Crypto.IProposedBlockSerializable): Promise<Buffer> {
+		const serializedLockProof = proposedBlock.lockProof
```

Comment:
It is probably the best to determine if lockProof is present instead writing empty data. It is going to be undefined in most cases.